tests/api-compatibility/README.md

```diff
@@ -0,0 +1,126 @@
+# Check-Api-Compatibility
+
+**Check-Api-Compatibility** is a process to check that changes are not causing
+APIs breakages on Mono.Android.dll
+The process uses [Microsoft.DotNet.ApiCompat][mdac] tool in order to
+test API compatibility between a two .NET assemblies
+When testing, the tool will compare a contract (old assembly) agaisnt an implementation (new assembly).
+
+The *contract* represents the API that's expected : for example a reference assembly or a previous version of an assembly. We could also called it the V1 assembly.
+
+The *implementation* represents the API that's provided : for example the current version of an assembly. We could also called it the V2 assembly.
+
+[mdac]: https://github.com/dotnet/arcade/tree/master/src/Microsoft.DotNet.ApiCompat
+
+## Build Task
+
+We have developed a build task that will wrap *Microsoft.DotNet.ApiCompat.exe* and perform required checks. The build task is called: *Xamarin.Android.Tools.BootstrapTasks.CheckApiCompatibility* and it is located under [BootstrapTasks][bst]
+The task will expect the following parameters:
+
+`ApiCompatPath` - Path where Microsoft.DotNet.ApiCompat nuget package is located.
+
+`ApiLevel` - Api level just built.
+
+`LastStableApiLevel` - The last stable api level.
+
+`TargetImplementationPath` - Path to the implementation assembly. Based on this path we will generate the contract path by replace current version with previous version
+
+`ApiCompatibilityPath` - Path to the Api Compatibility folder. On this folder we should find the acceptable-breakages files and the reference folder that contians the current Mono.Android.dll assembly.
+
+The *task* is aware of the Api levels and what is the previous Api level for agiven version, so When building api version 2, the Task knows if previous version is the version 1.5 or 1.0 for instance.
+
+## Usage
```

Comment:
Usage is both too much information and not enough information.

This isn't something for 3rd party use; it's only for *our* use.  As such, we don't need the `<Target/>` example, as that is "already" present in `src/Mono.Android/Mono.Android.csproj`/`src/Mono.Android/Mono.Android.targets`, meaning this comment will be invalidated/out-of-date as soon as we make changes to that call-site.

In some circumstances, "less is more."

On the other hand, there's not enough info here.  We can (should be!) "opinionated" about how things should work.  You mention `ApiCompatibilityPath`, but you don't mention *where that is within the tree*.  *That* part should be explicit.

Reply:
Fixed!